scripts/flatcar-tmpfiles

```diff
@@ -16,18 +16,23 @@ mkdir -p "${ROOT}/etc"
 
 # readable files
 umask 022
-if [[ ! -e "${ROOT}/etc/passwd" ]]; then
-    grep -E -e "^(${COPY_USERS}):" "${BASE}/passwd" > "${ROOT}/etc/passwd"
-fi
-if [[ ! -e "${ROOT}/etc/group" ]]; then
-    grep -E -e "^(${COPY_GROUPS}):" "${BASE}/group" > "${ROOT}/etc/group"
-fi
+# Output those lines in BASE/passwd (that are to be copied) when their user/group names are not in /etc/passwd already,
+# and append the entries from BASE/passwd to /etc/passwd.
+# But since we don't want lines/half lines being added also be used as patterns, we copy the file first to memory by
+# storing it in the PATTERNS variable.
+# (if it doesn't exist we are fine with empty output as patterns which means to match empty lines which inverted is all).
```

Comment:
```suggestion
# (We are fine if /etc/passwd doesn't exist as this will result in no patterns. As patterns have the filtering out function, no patterns mean that every line will match.)
```

Reply:
Thanks, applied it locally.